docs/kratos/manage-identities/25_import-user-accounts-identities.mdx

```diff
@@ -630,6 +630,87 @@ Connect ID Token provider such as Google.
 }
 ```
 
+### SAML connections
+
+:::note
+
+Importing SAML is only available on [some plans](https://www.ory.sh/pricing).
+
+:::
```

Comment:
```suggestion
```

IMO this is not needed. It is already obvious in the Ory Console and when you read the SAML specific config - it also does not really explain anything